src/components/DayPicker/DayPicker.tsx

```diff
@@ -126,7 +126,7 @@ export type DayPickerProps = Omit<TextFieldProps, 'value' | 'onChange'> & {
    *
    * Default is the main primary color of the theme.
    */
-  selectedBackgroundColor?: string;
+  pickerColor?: 'primary' | 'secondary';
```

Comment:
Can we update the JSDoc on this prop to be more explicit?

Something like `Default is 'primary'`

Reply:
✅